dashboard/config/locales/devise.en.yml

```diff
@@ -11,7 +11,8 @@ en:
       inactive: "Your account was not activated yet."
       invalid: "Invalid email or password."
       invalid_token: "Invalid authentication token."
-      locked: "Your account is locked."
+      locked: "Your account is locked. You will receive an email with instructions about how to unlock your account in a few minutes."
```

Comment:
Other things we could possibly do here:

- inform them that their account will be unlocked over time even if they don't click on the link (it's a configurable amount of time; default is one hour)
- link them to the "resend unlock instructions" form

Reply:
I think less detail is good here, even though people can read the source code or this PR to discover the timeout.